00-Getting_Started/README.md

```diff
@@ -0,0 +1,54 @@
+## 00 - Getting Started
+
+Before we get started with the exercises, we need to have all the elements
+in place. This directory has a couple of scripts to help you get quickly
+set up.
+
+This assumes that your machine has the necessary packages for using libvirt
+already installed, that your user has the right permissions and that you
+already have kubectl installed.  [Instructions on how to get there on
+Ubuntu or Debian](ubuntu.md)
+
+## Creating a Minikube cluster
+
+Install and run minikube by running `./start_minikube.sh`.  This will set
+up a minikube instance as described in our [instructions](../minikube.md).
+If it works fine, you'll have a cluster ready to use once the script is
+done. If there are errors, check the documentation.
+
+You can check that your cluster is running correctly by running `kubectl
+get pods -A` and verifying that you have the usual cluster management pods.
+
+**Note**: Minikube doesn't support nested virtualization, so if you want to
+run minikube inside a VM, you can do this by passing `--use-driver-none` to
```

Comment:
I think driver=none currently doesn't work with traceloop.

Reply:
I tested it and it works fine. But traceloop does take quite some memory, so the VM needs to have at least 2 GB of RAM. Added that to the docs.